### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by W72702** [aika-algorithm/aika-java#24]
Quick question how do you use this? I cant seem to figure it out and theres no documentation coming with the code.

**Comment by aika-algorithm** [comment]
Yes, I guess I've neglected the documentation in recent months and I will try to improve it soon. Currently, the project is on its way to the 2.0 release but is still very much in flux. Right now I'm mainly using the JacksonCookTest to stabilize the new binding-signal transition mechanism used for the linking process and to propagate the binding-signals from one activation to the next. This test case might therefore be a good starting point.
